mct.json

```diff
@@ -5,7 +5,8 @@
     "requiredDate": "2020-08-01",
     "downloadUrl": "https://portal.mopria.org/DesktopModules/Inventures_Document/FileDownload.aspx?ContentID=4252",
     "issuesVersion": "v1",
-    "issuesUrl": "https://portal.mopria.org/MyWorkGroups/Documents.aspx?Type=Detail&DocumentID=4569"
+    "issuesUrl": "https://portal.mopria.org/MyWorkGroups/Documents.aspx?Type=Detail&DocumentID=4569",
+    "certificationExpired": ["printer-v1.3"]
```

Comment:
certificationsExpired

Reply:
As discussed, lets change this to specifications : { print1.3 : "audit" ,  print1.4 : reference }  etc